src/ruby/yast/rspec/scr.rb

```diff
@@ -0,0 +1,96 @@
+require "rspec"
+require "yast"
+
+module Yast
+  module RSpec
+    # RSpec extension to handle several agent operations.
+    module SCR
+      # Shortcut for generating Yast::Path objects
+      #
+      # @param route [String] textual representation of the path
+      # @return [Yast::Path] the corresponding Path object
+      def path(route)
+        Yast::Path.new(route)
+      end
+
+      # Encapsulates SCR calls into a chroot.
+      #
+      # If a block if given, the SCR calls in the block are executed in the
+      # chroot and the corresponding SCR instance is automatically closed when
+      # the block ends (or if an exception is raised by the block).
+      #
+      # If a block is not given, the chroot must be explicitly closed calling
+      # reset_root_path.
+      #
+      # Nesting of chroots is forbidden and the method will raise an exception
+      # if is called without closing a previous chroot.
+      #
+      # @param directory [#to_s] directory to use as '/' for SCR calls
+      #
+      # @example Usage with a block
+      #   change_scr_root("/home/chroot1") do
+      #     # This reads the content of /home/chroot1/
+      #     Yast::SCR.Read(path(".target.dir"), "/")
+      #   end
```

Comment:
Nice, I think we should also add example with rspec like:
```
around(:all) { |example| change_scr_root("/home/chroot1", &example) }
```
this way it is safe and easy to use :)